### PR TITLE
PMM-6377 mnogo_exporter packaging

### PR DESCRIPTION
**Pull request by Sevlin**
Added **mnogo_exporter** for pmm2 client.

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1038-1af396a
client docker - perconalab/pmm-client-fb:PR-1038-1af396a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1038-1af396a.tar.gz

File: build/rpm/pmm2-client.spec [hunk 1]
```diff
@@ -75,6 +75,7 @@ install -m 0755 bin/postgres_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exp
 install -m 0755 bin/mongodb_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
 install -m 0755 bin/proxysql_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
 install -m 0755 bin/rds_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
+install -m 0755 bin/mnogo_exporter $RPM_BUILD_ROOT/usr/local/percona/pmm2/exporters
```
Comment (EvgeniyPatlan) [inline build/rpm/pmm2-client.spec:78]: @alexanderblinkov-percona  are we going to deliver both versions of mongodb_exporter?

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1038-2e9b8b2
client docker - perconalab/pmm-client-fb:PR-1038-2e9b8b2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1038-2e9b8b2.tar.gz

**Comment by EvgeniyPatlan** [comment]
will be reworked

**Pull request closed without merging**